src/test/java/com/hpe/adm/octane/ideplugins/unittests/ConnectionSettingsUTCase.java

```diff
@@ -21,120 +21,67 @@
 
     private final Logger logger = LogManager.getLogger(IntegrationTestBase.class.getName().toString());
 
-    @Test
-    public void validateGoodCredentials() {
-        ConnectionSettings connectionSettings = connectionSettingsProvider.getConnectionSettings();
-        OctaneHttpClient octaneHttpClient = new GoogleHttpClient(connectionSettings.getBaseUrl());
-        EntityModel entityModel = getCurrentUser();
-        try {
-            assert octaneHttpClient.authenticate(new SimpleUserAuthentication(entityModel.getValue("email").getValue().toString(), "Welcome1", ClientType.HPE_MQM_UI.name()));
-        } catch (OctaneException e) {
-            String errorMessage = SdkUtil.getMessageFromOctaneException(e);
-            logger.error(errorMessage);
-        }
-    }
+    private TestService testService = new TestService();
 
-    @Test
-    public void validateGoodUserWrongPasswordCredentials() {
-        ConnectionSettings connectionSettings = connectionSettingsProvider.getConnectionSettings();
-        OctaneHttpClient octaneHttpClient = new GoogleHttpClient(connectionSettings.getBaseUrl());
-        String username = getCurrentUser().getValue("email").getValue().toString();
-        String password = UUID.randomUUID().toString();
-        try {
-            assert !octaneHttpClient.authenticate(new SimpleUserAuthentication(username, password, ClientType.HPE_MQM_UI.name()));
 
-        } catch (OctaneException e) {
-            String errorMessage = SdkUtil.getMessageFromOctaneException(e);
-            logger.error(errorMessage);
-        }
-    }
+    private boolean validateCredentials(String username, String password, String baseUrl) {
 
-    @Test
-    public void validateWrongUserGoodPasswordCredentials() {
-        ConnectionSettings connectionSettings = connectionSettingsProvider.getConnectionSettings();
-        OctaneHttpClient octaneHttpClient = new GoogleHttpClient(connectionSettings.getBaseUrl());
-        String username = UUID.randomUUID().toString();
-        String password = "Welcome1"; //--default password
+        OctaneHttpClient octaneHttpClient = new GoogleHttpClient(baseUrl);
         try {
-            assert !octaneHttpClient.authenticate(new SimpleUserAuthentication(username, password, ClientType.HPE_MQM_UI.name()));
-
+            return octaneHttpClient.authenticate(new SimpleUserAuthentication(username, password, ClientType.HPE_MQM_UI.name()));
         } catch (OctaneException e) {
             String errorMessage = SdkUtil.getMessageFromOctaneException(e);
             logger.error(errorMessage);
+            return false;
         }
     }
 
     @Test
-    public void validateWrongUserWrongCredentialsCredentials() {
+    public void testValidateCredentials() {
```

Comment:
it is good that you reuse the same code, but split this into separate tests and give them relevant names for each scenario covered